### DISCUSSION TITLE
support for wildcard virtualhost fqdn (*)

### DISCUSSION THREAD

**Discussion by puneri78** [projectcontour/contour#5571] [Q&A]
How to support wildcard virtualhost fqdn i.e. any fqdn support which is deployed in cloud behind ALB ?
e.g. two ALBs with following DNS mapping - to use same envoy.
example.com
example.org

Anything else you would like to add:

What worked is following - but like to know other opinions

As of now I'm able to achieve this and setting fqdn virtualhost to "*" - by disabling following in FilterMisdirectedRequests
// if host ~= target then
// request_handle:respond(
// {[":status"] = "421"},
// string.format("misdirected request to %%q", host)
// )
// end

though this works just fine for usecase I have, like to know if this is valid feature request and we can add some kind of flag/header similar to authority header to support this usecase.
Environment:

Contour version: 1.25
-----------
https://github.com/projectcontour/contour/issues/5554 (also created this as issue)

**Comment by skriss** [comment]
@puneri78 if you are looking to expose the same set of routes under two different fqdn's, then the approach described in https://projectcontour.io/docs/1.25/config/virtual-hosts/#virtualhost-aliases may work for you
